doc/nrf/app_dev/device_guides/nrf53/building_nrf53.rst

```diff
@@ -133,9 +128,6 @@ To program the nRF5340 DK from the command line, use either west (which uses nrf
 
                west flash
 
-            .. note::
-                 The minimum supported version of nrfjprog for multi-image builds for different cores is 10.21.0.
-
          .. group-tab:: nRF Util
 
             Enter the following commands to program multiple image builds for different cores:
```

Comment:
@carlescufi , given the west runner update, can we simplify this section to only mention west flash, without explicit mention of nRF Util commands? Note nRF Util commands in lines 145-146 and 88, 102, 111.